3-Authorization-II/protect-web-api/README.md

```diff
@@ -1,226 +1,230 @@
 ---
 page_type: sample
-languages:
+name: Enable your Java Spring Boot web app to sign in users with Azure Active Directory and call a protected Web api
+description: Enable your Java Spring Boot web app to sign in users in your Azure AD tenant and call a protected Web api protected with Spring Security
+- languages:
   - java
 products:
-  - azure
+    - java
```

Comment:
Need to double check if these products are part of the [taxonomy](https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main) (`java `and `spring-boot` are most probably not, you should add `msal-java`)